.github/workflows/integrations.yml

```diff
@@ -382,3 +382,49 @@ jobs:
         with:
           slack-webhook: ${{ secrets.E2E_INTEGRATIONS_SLACK_WEBHOOK }}
           job-status: ${{ needs.detectron2.result }}
+
+  rlang:
+    env:
+      NEPTUNE_PROJECT: e2e-tests/integrations
+    timeout-minutes: 75
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: macos-latest,   r: 'release'}
+          - {os: ubuntu-latest,  r: 'release'}
+    runs-on: ${{ matrix.os }}
+    name: 'R (${{ matrix.os }})'
```

Comment:
Underlying miniconda action can handle different Python versions:
```
      - uses: conda-incubator/setup-miniconda@v2
        with:
          auto-update-conda: true
          python-version: ${{ matrix.python-version }}
```
Could we pass it through and test on 3.7 + 3.9/3.10?

Reply:
We could, but do we need it? Usually, unit tests are not for testing the dependencies, and this would only run the client code using different Python versions, which should already be tested by the unit tests for the client itself. This integration only calls the public API of the client, so if the public API works, we don't need to recheck it.

Reply:
Ok. I buy it